scripts/nuget-debug-helpers.ps1

```diff
@@ -1,6 +1,6 @@
 $NuGetClientRoot= Resolve-Path $(Join-Path $PSScriptRoot "..\")
 
-$VSVersion = "16.0"
```

Comment:
Can we make it read the env var $env:VisualStudioVersion if it's set? 
That way if you are using a specific developer command prompt it'll just use those defaults. 

I am still working on dev16 as well, so reading the env var would allow switching between different environments seemless.

Reply:
Done!